config/default.py

```diff
@@ -37,6 +37,7 @@ class Default:
 
     GENMEDIA_BUCKET: str = os.environ.get("GENMEDIA_BUCKET")
     PUBLIC_BUCKET: bool = os.environ.get("PUBLIC_BUCKET", "False").lower() in ("true", "1")
+    SHOW_RESULTS_PAUSE_TIME: int = os.environ.get("SHOW_RESULTS_PAUSE_TIME", 1)
```

Comment:
I think we need to cast to `int` as os.environ.get will always return a string.

Reply:
Added a cast, ptal